stock_inventory_revaluation/README.rst

```diff
@@ -0,0 +1,88 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+===================================
+Stock Account Inventory Revaluation
+===================================
+
+If your company runs a perpetual inventory system, you may need to perform
+inventory revaluation.
+
+You can re-valuate inventory values by:
+
+* Changing the inventory valuation of a specific product. The cost price
+  is changed, and the inventory value is recalculated according to the new
+  price. In case of real price, you can select on which quants you want to
+  change the unit cost.
+
+* Changing the value of the inventory. The quantity of inventory remains
+  unchanged, resulting in a change in the price.
+
+
+Configuration
+=============
+
+* Go to *Inventory / Configuration / Products / Product Categories* and
+  define, for each category, a Valuation Increase Account and a Valuation
+  Decrease Account. These accounts will be used as contra-accounts to the
+  Stock Valuation Account during the inventory re-valuation.
+
+* Users willing to access to the Inventory Revaluation menu should be
+  members of the group "Manage Inventory Valuation and Costing Methods".
+
+
+Usage
+=====
+
+* Go to *Inventory / Inventory Control / Inventory Revaluation / Products*
```

Comment:
It will show once you apply "Manage Inventory Valuation and Costing Methods" to the user